src/test/java/bio/terra/service/dataset/ValidatorTestUtils.java

```diff
@@ -12,7 +12,7 @@
 
 public class ValidatorTestUtils {
 
-  public static void checkValidationErrorModel(ErrorModel errorModel, String[] messageCodes) {
+  public static void checkValidationErrorModel(ErrorModel errorModel, String... messageCodes) {
```

Comment:
This change allows callers to pass in the list of strings directly instead of creating an array to pass in.